### PR TITLE
kola: add NFS version 4 test

### PR DESCRIPTION
**Pull request by mischief**

### CODE REVIEW

File: kola/tests/misc/nfs.go [hunk 1]
```diff
@@ -61,7 +55,23 @@ func NFS(c platform.TestCluster) error {
...
+After=network-online.target
+Requires=network-online.target
+After=rpc-statd.service
+Requires=rpc-statd.service
+
+[Mount]
+What=%s:/tmp
+Where=/mnt
+Type=%s
+Options=defaults,noexec
```
Comment (marineam) [inline kola/tests/misc/nfs.go]: You need to use the `nfs` type and toggle `nfsvers=4` vs `nfsvers=3` in options. The client and server are free to negotiate version 3 or 4 if with the `nfs` type.

Comment (marineam) [inline kola/tests/misc/nfs.go]: So in short, you may have been testing v4 all along ;-)

Comment (mischief) [inline kola/tests/misc/nfs.go]: i'm not so sure - NFSv4 is failing against current alpha, but not against my local copy with updated nfs-utils. :confounded:

**Comment by pbx0** [comment]
LGTM

File: kola/tests/misc/nfs.go (hunk 1 above)
Comment (marineam) [inline kola/tests/misc/nfs.go]: may have been negotiating v3 then since v4 was broken. *shrug*

Comment (marineam) [inline kola/tests/misc/nfs.go]: whatever the case, nfsvers= is required to be sure of the version in use.

**Comment by mischief** [comment]
@marineam ptal

**Comment by marineam** [comment]
LGTM

**Pull request merged**